kong/cmd/restart.lua

```diff
@@ -20,7 +20,10 @@ local function execute(args)
 
   log.enable()
 
-  pcall(stop.execute, args, { quiet = true })
+  local ok = pcall(stop.execute, args, { quiet = true })
+  if not ok then
+    log.enable()
+  end
```

Comment:
`log.enable()` is idempotent, so we could call it unconditionally, we don't have to check for `pcall()`'s return value. In fact, given `quiet = true` will call `log.disable` in `kong stop`'s `execute()` method, we should move the above `log.enable()` on L.21 down here instead, since it is a no-op where it is above right now.